.changeset/rare-toys-wave.md

```diff
@@ -0,0 +1,9 @@
+---
+"wrangler": minor
+---
+
+Add command for fetching R2 Event Notification configurations for a given bucket
```

Comment:
Can you make this changeset conform to the format in CONTRIBUTING.md please

Reply:
What specifically needs added? Just `feature:` at the beginning?

Reply:
Yep....

```suggestion
feature: add command for fetching R2 Event Notification configurations for a given bucket
```

Reply:
Got it, done